### PR TITLE
issue-1872: Upgrade the maven-jar-plugin from 3.0.2 to 3.2.0

### PR DESCRIPTION
**Pull request by yuhanzz**
Resolved strongbox/strongbox#1872
Passed all the tests following "Steps For Upgrading Dependencies And Plugins".

### CODE REVIEW

**Comment by carlspring** [comment]
Thanks! Have you also looked into the part regarding adding the pull for the `strongbox` project as well?

**Comment by yuhanzz** [comment]
> Thanks! Have you also looked into the part regarding adding the pull for the `strongbox` project as well?

Hi, I found that I ignored several poms to update when I was going to add PR to StrongBox. Now I've updated all the poms and redone all the tests. The PR is strongbox/strongbox#1885. Thanks!

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-issue-1872-SNAPSHOT</version>
```
Comment (carlspring) [inline pom.xml:9]:
```suggestion
    <version>1.0-SNAPSHOT</version>
```
Reverting back to `1.0-SNAPSHOT`, as this has now passed in Jenkins! :)

**Comment by carlspring** [comment]
Thank you for your contribution! :)

**Pull request merged**